src/mbgl/map/transform.cpp

```diff
@@ -576,4 +578,16 @@ LatLng Transform::screenCoordinateToLatLng(const ScreenCoordinate& point, LatLng
     return state.screenCoordinateToLatLng(flippedPoint, wrapMode);
 }
 
+double Transform::getMaxPitchForEdgeInsets(const EdgeInsets &insets) const
+{
+    double centerOffsetY = 0.5 * (insets.top() - insets.bottom()); // See TransformState::getCenterOffset.
+
+    const auto height = state.size.height;
+    assert(height);
```

Comment:
how is that guarantied?

Reply:
This is invalid state as it would result by division by zero bellow (old and new code).
In production code it couldn't happen:
Map::Impl::Impl calls [Transform::resize](https://github.com/mapbox/mapbox-gl-native/blob/e5431d01817c0546aa0b971a7b0b33c9bbd87077/src/mbgl/map/transform.cpp#L49), where we ensure height != 0.
Constructed value is invalid (0) and could happen with unit tests constructing Transform with no resize call. Fixed some those tests in this patch and assert is added here...